devtools/cli/src/main/java/io/quarkus/cli/plugin/PluginCatalogService.java

```diff
@@ -0,0 +1,97 @@
+package io.quarkus.cli.plugin;
+
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import io.quarkus.cli.common.OutputOptionMixin;
+import picocli.CommandLine;
+
+public class PluginCatalogService extends CatalogService<PluginCatalog> {
+
+    public PluginCatalogService() {
+        super(PluginCatalog.class, GIT_ROOT,
+                p -> p.resolve(".quarkus").resolve("cli").resolve("plugins").resolve("catalog.json"));
```

Comment:
we have so many things called catalog so would prefer other name or prefixed somehow. i.e. quarkus-cli-catalog.json